packages/grafana-prometheus/src/result_transformer.test.ts

```diff
@@ -261,11 +261,11 @@ describe('Prometheus Result Transformer', () => {
           createDataFrame({
             refId: 'A',
             fields: [
-              { name: 'time', type: FieldType.time, values: [6, 5, 4] },
+              { name: 'time', type: FieldType.time, values: [4, 5, 6] },
```

Comment:
not technically necessary, but iiuc prometheus responses are always ordered ASC within each data frame, so this should be more representative of real datasets.